client/src/templates/Challenges/components/Challenge-Description.js

```diff
@@ -9,11 +9,16 @@ const propTypes = {
   section: PropTypes.string
 };
 
+function emptyInstruction(instructions) {
+  return (/^<section\s+id\s*=\s*["|']instructions["|']\s*>\s*<\/section>$/)
+  .test(instructions);
+}
+
 function ChallengeDescription({ description, instructions, section }) {
   return (
     <div className={`challenge-instructions ${section}`}>
       <div dangerouslySetInnerHTML={{ __html: description }} />
-      {instructions ? (
+      {!emptyInstruction(instructions) ? (
```

Comment:
```suggestion
      {!emptyInstruction(instructions) && (
```

and remove `<hr />` at lines 25 and 28. and add `<hr />` before the last `</div>`